server/athenian/api/serialization.py

```diff
@@ -90,37 +89,52 @@ def _deserialize_object(value: T) -> T:
     return value
 
 
-def deserialize_date(string: str) -> datetime.date:
-    """Deserializes string to date.
+# default bounds for deserialize_date and deserialize_datetime
+_DATETIME_MIN = datetime(2000, 1, 1)
+_DATE_MIN = _DATETIME_MIN.date()
+_DATETIME_MAX_FUTURE = timedelta(days=365)
```

Comment:
This suggests a datetime but the type is timedelta, let's rename to _?_MAX_FUTURE

Reply:
renamed as `_DATETIME_MAX_FUTURE_DELTA`, which is "maximum time delta in the future for datetime deserialization"